operations/commit_queue.go

```diff
@@ -10,7 +11,13 @@ import (
 	"github.com/urfave/cli"
 )
 
-const itemFlagName = "item"
+const (
+	itemFlagName       = "item"
+	refFlagName        = "ref"
+	pauseFlagName      = "pause"
+	identifierFlagName = "identifier"
+	commitQueueAlias   = "__commit_queue"
```

Comment:
with this alias the eventual version will be populated with the variants/tasks configured for the project's commit queue.

Reply:
done. In globals.go